src/objects/Contact.object

```diff
@@ -19,6 +19,18 @@
         <type>Email</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Chosen_Full_Name__c</fullName>
+        <description>The full name that the Contact prefers to use instead of a legal or official name.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The full name that the Contact prefers to use instead of a legal or official name.</inlineHelpText>
```

Comment:
If we retrieve the metadata from the org, there should be corresponding translation files with changes as well.

Reply:
Yep good call. I added this manually via the XML...I bet I can add that manually as well to the necessary files.

Reply:
If not, I will pull.